.github/CONTRIBUTING.md

```diff
@@ -78,6 +78,30 @@ The following guidelines may answer most of your questions and help you get star
         cp guides/v2.2/install-gde/composer.md guides/v2.3/install-gde/composer.md
         ```
 
+    - When you create a new topic (a new `.md` file will be created) and its content is the same for 2.2 and 2.3 versions, please create a symbolic link.
+
+        For example: you created a new file for v.2.2 - `guides/v2.2/install-gde/new-doc-topic.md` and its content is the same for v.2.3 then
+        - do not create a copy of `guides/v2.2/install-gde/new-doc-topic.md` in the `guides/v2.3/install-gde/` folder
+        - use the command below to create the appropriate symbolic link
+
+        ```bash
+        cd <DEVDOCS_REPOSITORY_ROOT_DIR>
+        ```
+
+        ```bash
+        ln -s guides/v2.2/install-gde/new-doc-topic.md guides/v2.3/install-gde/new-doc-topic.md
+        ```
+
+    - This approach works for images as well. So if you used an image for v.2.2 and that image is the same for v.2.3 then create a symbolic link and do not make a copy:
```

Comment:
```suggestion
    - Because this approach works for images as well, if you used an image for v.2.2 and that image is the same for v.2.3, create a symbolic link and do not make a copy:
```

Reply:
**fixed**